test/python/tokenizer/sanitizers/test_sanitizer_config.py

```diff
@@ -78,36 +74,61 @@ def test_create_split_regex_empty_delimiter():
         regex = SanitizerConfig({'delimiters': ''}).get_delimiter()
 
 
-@pytest.mark.parametrize('inp', ('name', 'name:de', 'na\\me', '.*'))
-def test_create_kind_filter_no_params(inp):
-    filt = SanitizerConfig().get_filter_kind()
+@pytest.mark.parametrize('inp', ('name', 'name:de', 'na\\me', '.*', None))
+def test_create_name_filter_no_param_no_default(inp):
+    filt = SanitizerConfig({'filter-kind': 'place'}).get_filter('name')
 
     assert filt(inp)
 
 
 @pytest.mark.parametrize('kind', ('de', 'name:de', 'ende'))
+def test_create_kind_filter_default_positive(kind):
+    filt = SanitizerConfig({'name': 'abc'}).get_filter('filter-kind', ['.*de'])
+
+    assert filt(kind)
+
+@pytest.mark.parametrize('kind', ('de', 'name:de', 'ende', None))
+def test_create_kind_filter_default_negetive(kind):
+    filt = SanitizerConfig().get_filter('filter-kind', ['.*fr'])
+
+    assert not filt(kind)
+
+
+@pytest.mark.parametrize("kind", ("lang", "lang:de", "langxx"))
 def test_create_kind_filter_custom_regex_positive(kind):
-    filt = SanitizerConfig({'filter-kind': '.*de'}).get_filter_kind()
+    filt = SanitizerConfig(
+        {
+            "name": "abc",
+            "filter-kind": "lang.*",
+        }
+    ).get_filter("filter-kind", [".*fr"])
 
     assert filt(kind)
 
 
-@pytest.mark.parametrize('kind', ('de ', '123', '', 'bedece'))
+@pytest.mark.parametrize("kind", ("de ", "123", "", "bedece", None))
```

Comment:
Please go with the existing quoting style.

The convention for quotes is:
 * use single quotes for all simple strings unless the string contains a single string (so, it should be 'London' but "London's subway")
 * use double quotes for format strings (i.e. `f"London {foo}"`)

I know that this convention is not followed everywhere but we should at least get it consistent in new code.